src/N3DataFactory.js

```diff
@@ -256,6 +265,53 @@ export class Quad {
 }
 export { Quad as Triple };
 
+// QuadTerm constructor
+export class QuadTerm extends Term {
+  constructor(quad) {
+    super(`<<${quadToId(quad)}>>`);
+    this.quad = quad;
+  }
+
+  // ### The term type of this term
+  get termType() {
+    return 'QuadTerm';
+  }
+
+  // ### The text value of this literal
+  get value() {
+    return this.quad;
+  }
+}
+
+// ### Creates a string from a Quad
+function quadToId(quad) {
+  let result = [quad.subject, quad.predicate, quad.object].map(termToId).map(escape).join(' ');
+
+  // Omit default graph
+  if (!isDefaultGraph(quad.graph)) {
+    result += ` ${termToId(quad.graph)}`;
```

Comment:
…and would add this to the single string interpolation then (hoping that it does not become too long of a line, but we could add returns inside of `${…}`)